privacy-configuration/config4_reference.json

```diff
@@ -0,0 +1,60 @@
+{
+    "readme": "This configuration has only tracking link features",
+    "features": {
+        "trackingLinks": {
+            "state": "enabled",
+            "exceptions": [
+                {
+                    "domain": "example2.com",
+                    "reason": "Site breakage"
+                }
+            ],
+            "settings": {
+                "ampLinkFormats": [
+                    "https?:\\/\\/(?:w{3}\\.)?google\\.\\w{2,}\\/amp\\/s\\/(\\S+)",
```

Comment:
those regexes are not anchored (^$) and will match things like 'https://bla.com/https://www.google.com/amp/s/www.example.com/some/article'. Is that expected? If we expect implementations to provide $ and ^ then we need a test for it (like "https://bla.com/https://www.google.com/amp/s/www.example.com/some/article" that should NOT match)

Reply:
also, this doesn't match google.co.uk - not sure if that's expected?